### DISCUSSION TITLE
Deployment guide for scully

### DISCUSSION THREAD

**Discussion by tamirazrab** [scullyio/scully#1565] [Q&A] [answered]
I actually want to know how you deploy it to your own custom server aside from netlify.  I actually deployed it to netlify and backend guy takes build and deploys it however there is so many problems we're facing, in our own deployment but in netlify all seems to be working as expected. There's seems to be problem in how we're deploying, how should we deploy it - or how netlify deploys it under the hood?

**Comment by SanderElias** [comment, marked as answer]
When you did run Scully, the output is in the `./dist/static` folder (unless you changed settings).

For all deployment purposes, this is now an old-fashioned static site, also known as 'Multi-Page-Application'. 
That means you can copy it to the root of any static-site hosting provider.  Most of them will work without further settings.

Although some newer providers now take the SPA first approach. Then you need to configure them to use an MPA instead.
Also, you might want to add 404 handling to the server. It needs to be set up so, that when the asked for the file is not found, it loads the main index.html instead (or the 404.html, which is the untouched application)
